internal/guid/guid.go

```diff
@@ -2,8 +2,11 @@ package guid
 
 import (
 	"crypto/rand"
+	"encoding/json"
 	"fmt"
 	"io"
+	"strconv"
+	"strings"
 )
 
 type GUID [16]byte

```

Comment:
The more I think about it we should use `github.com/google/uuid`. It does all the things here we need and then we dont need to maintain this class at all right?